admin/services/check-form.service.js

```diff
@@ -176,6 +176,19 @@ const checkFormService = {
       }
     })
     return canDelete
+  },
+  /**
+   * Build a form name based on the file name.
+   * @param fileName
+   */
+  buildFormName: async (fileName) => {
+    const minFileNameSize = 4 // for example: a.csv
```

Comment:
As this is a `length` the example filename will have a length of 5, but code disagrees and says 4. Shouldn't they agree?